include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -210,6 +210,70 @@ struct BFloat16_t {
 
 static_assert(sizeof(BFloat16_t) == sizeof(uint16_t), "Sizes must match");
 
+/** \brief float8e4m3fn (Float8 Floating Point) data type
+ * \details It is necessary for type dispatching to make use of C++ API
+ * The type is implicitly convertible to/from uint8_t.
```

Comment:
Some docs on what the types are, and may be refs to more information.

Reply:
It is ok to use a link to onnx documentation?

Reply:
I think so.

Reply:
done